source/deployment-options/docker/wazuh-container.rst

```diff
@@ -44,7 +44,7 @@ Single-node Deployment
         
       .. code-block:: yaml
         
-         # Wazuh App Copyright (C) 2021 Wazuh Inc. (License GPLv2)
+         # Wazuh dashboard Copyright (C) 2021 Wazuh Inc. (License GPLv2)
```

Comment:
This line is coming from a file of Docker deployment that is defined on the `wazuh-docker` repository. Regarding the context, it seems to be related to generates the certificates to using for Wazuh dashboard, Wazuh indexer and Wazuh server. It is not clear to me the usage of `App` term on this context.

https://github.com/wazuh/wazuh-docker/blob/v4.8.0-beta6/single-node/generate-indexer-certs.yml#L1

Note the year on the documentation is different to the `wazuh-docker` repository.

Reply:
Fixed